biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain-class.ts

```diff
@@ -82,13 +73,13 @@ export class BioBrain implements Brain {
     // @see https://github.com/googleinterns/step141-2020/issues/54
     allGridItems.map(item => {
       const gridItem = this.clonedGraph.node(item);
-      if (gridItem.name.includes(GRID_ITEM_NAMES.ENERGY_USER)) {
+      if (gridItem.gridItemName.includes(GRID_ITEM_NAMES.ENERGY_USER)) {
```

Comment:
Will gridItem.gridItemName == GRID_ITEM_NAMES.ENERGY_USER?  If so, make these == instead of .includes()

Reply:
> Will gridItem.gridItemName == GRID_ITEM_NAMES.ENERGY_USER? If so, make these == instead of .includes()

It does not equal to. For instance if there are two energy users(buildings), their gridItemNames are different. However, they both start with `GRID_ITEM_NAMES.ENERGY_USER`

Reply:
👍